### PR TITLE
Add recent changes to vellum deploy script

### PR DESCRIPTION
**Pull request by emord**
@millerdev

### CODE REVIEW

File: scripts/vellum-to-hq [hunk 1]
```diff
@@ -94,9 +95,21 @@ def build_vellum(path, branch, test=True):
     return str(git("rev-parse", "HEAD"))
 
 
+def get_old_vellum_rev(git):
+    commit = str(git('log', '--oneline', '--grep', 'vellum', '-n', '1'))
```
Comment (millerdev) [inline scripts/vellum-to-hq]: might want to change the grep to `'vellum:'`

Comment (emord) [inline scripts/vellum-to-hq]: Fixed

**Comment by millerdev** [comment]
I approve of this merge

**Comment by emord** [comment]
yeah didn't mean to merge. accidental button press

**Pull request merged**